src/main/java/org/sonatype/goodies/packageurl/PackageUrlBuilder.java

```diff
@@ -145,7 +159,7 @@ public PackageUrl build() {
    *
    * @since 1.0.1
    */
-  PackageUrl buildAndValidate(final boolean validate) {
+  PackageUrl buildAndValidate(final boolean validate, final boolean correct) {
```

Comment:
see above, I think this can be logical configuration on the builder instance and doesn't need to be exposed on the api?

Reply:
Done